### DISCUSSION TITLE
where should modified footer.html be placed in layouts?

### DISCUSSION THREAD

**Discussion by bgriffen** [adityatelange/hugo-PaperMod#586] [Q&A]
I tried partials, _default and the top level but it isn't overriding - does this get picked up if edited?

Ty.

**Comment by adityatelange** [comment]
Read this
https://adityatelange.github.io/hugo-PaperMod/posts/papermod/papermod-faq/#override-theme-template

**Reply by bgriffen** [reply]
I read all of the docs before I posted my query. That's why I said 

> I tried partials, _default and the top level but it isn't overriding

which is what the doc link suggested doing. It's OK, I'll work it out.
